lessons/pydata/pandas_core/index.ipynb

```diff
@@ -0,0 +1,3402 @@
+{
```

Comment:
<p><code>dtype('O')</code> je trochu nešťastný, nepoužijeme tady <code>series</code> (tj <code>int</code>)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Souhlasím, nebo vysvětlit, co to vlastně znamená (já to třeba narozdíl od int nevím)

Reply:
Ok, ok...

Reply:
<p>Hodil jsem tam místo toho vysky.</p>